common/build.gradle

```diff
@@ -1,10 +1,13 @@
-
-
 plugins {
     id "org.sonarqube" version "2.5"
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'pmd'
+apply plugin: 'checkstyle'
+apply plugin: 'jacoco'
```

Comment:
I don't recall testing `jacoco` in this project yet. These numbers would be good to have, please give this a run and make sure the generated `jacoco exec` files are readable and create working reports.

Reply:
Added jacoco task and verified the exec file